content/docs/quickstart/_index.md

```diff
@@ -7,6 +7,41 @@ weight: 1
 hidechildren: true # this flag hides all sub pages in the sidebar-multicard.html
 ---
 
+We'll run Keptn on a local [k3d](https://k3d.io) cluster.
+
+## Prerequisites
+- [Docker](https://docker.com/)
+- [k3d](https://k3d.io) (find [installation instructions](https://k3d.io/#install-script) or run<br> `wget -q -O - https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash` )
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [helm](https://helm.sh/) when running the demos in step 4
+
+## Install Keptn
+
+1. Create local k3d cluster
+
+    ```
+    k3d cluster create mykeptn -p "8082:80@agent[0]" -p "8443:443@agent[0]" --k3s-server-arg '--no-deploy=traefik' --agents 1
```

Comment:
Please include the installation instructions for k3d here, e.g.:
```
wget -q -O - https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
k3d cluster create mykeptn -p "8082:80@agent[0]" -p "8443:443@agent[0]" --k3s-server-arg '--no-deploy=traefik' --agents 1
```
Also, please use a fixed version of k3d.

Reply:
done